packages/code-du-travail-frontend/src/outils/common/situations.utils.js

```diff
@@ -86,7 +87,9 @@ export function getSituationsFor(data, obj) {
 
 export const isNotYetProcessed = (data, idcc) => {
   const situtation = data.filter(
-    (situation) => parseInt(situation.idcc, 10) === parseInt(idcc, 10)
+    (situation) =>
+      Object.keys(situation.criteria).length &&
```

Comment:
je sais pas si c'est le meilleur moyen de tester qu'un objet n'est pas vide {} ?

Reply:
Ah oui, en vrai ce que tu fais ça passe

Pourquoi pas le mettre dans une fonction ?

```suggestion
function isObjectEmpty(obj) {
    return Object.keys(obj).length === 0;
}
```

Reply:
done 👍